angular/src/lib/slider/slider.component.ts

```diff
@@ -0,0 +1,240 @@
+import {
+  AfterViewInit,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  EventEmitter,
+  forwardRef,
+  Input,
+  OnInit,
+  Output,
+  ViewChild,
+} from '@angular/core';
+import { NG_VALUE_ACCESSOR, ControlValueAccessor } from '@angular/forms';
+import { SliderPointerChange } from './slider-pointer.component';
+
+const CUI_SLIDER_VALUE_ACCESSOR: any = {
```

Comment:
Can you use the type `Provider` here instead of `any`?
```suggestion
import { Provider } from '@angular/core';
const CUI_SLIDER_VALUE_ACCESSOR: Provider = {
```

Reply:
updated